### PR TITLE
PWA-2664: Swatches Components - Base Styles 

### PR DESCRIPTION
**Pull request by rwalter215**
## Description

- Update Swatch & SwatchList style to reflect Base theme
- Update Tile & TileList style to reflect Base theme
- Update relevant unit tests/snapshots

## Related Issue

Closes [PWA-2664](https://jira.corp.magento.com/browse/PWA-2664).

## Acceptance
PM & UX Approval: Not approved yet

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: packages/peregrine/lib/ui/components/ProductOptions/swatch.module.css [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-    --venia-swatch-bg: var(--venia-global-color-gray-100);
+.base {
+    --venia-swatch-bg: rgb(var(--color-neutral-200));
+    composes: root from '../clickable.module.css';
+
+    composes: rounded-full from global;
+    composes: border-buttonWidth from global;
+    composes: border-solid from global;
+    composes: h-[2.5rem] from global;
+    composes: w-[2.5rem] from global;
```
Comment (anthoula) [inline packages/peregrine/lib/ui/components/ProductOptions/swatch.module.css:9]: Would the Tailwind basic usage work?

```
composes: h-10 from global;
composes: w-10 from global;
```

**Review by anthoula — CHANGES_REQUESTED** [review]

File: packages/peregrine/lib/ui/components/ProductOptions/swatch.module.css (hunk 1 above)
Comment (anthoula) [reply packages/peregrine/lib/ui/components/ProductOptions/swatch.module.css]: Can you check the PR for more instances that can use Tailwind's basic usage values? Thank you

**Comment by glo42707** [comment]
run cypress

**Comment by glo42707** [comment]
run lighthouse-mobile

**Comment by glo42707** [comment]
run lighthouse-desktop

**Comment by glo42707** [comment]
run pr-deploy

**Comment by glo42707** [comment]
run cypress

**Pull request closed without merging**